packages/react-liveness/src/components/FaceLivenessDetector/service/utils/createStreamingClient/CustomWebSocketFetchHandler.ts

```diff
@@ -159,14 +159,9 @@ export class CustomWebSocketFetchHandler {
     socket: WebSocket,
     connectionTimeout: number
   ): Promise<void> {
-    return new Promise((resolve, reject) => {
+    return new Promise((resolve) => {
       const timeout = setTimeout(() => {
         this.removeNotUsableSockets(socket.url);
-        reject({
```

Comment:
Not sure we want to fully remove the reject? Did we try just removing the metadata? Does the error still come through?